zipkin-anormdb/src/main/scala/com/twitter/zipkin/util/DBConfig.scala

```diff
@@ -90,29 +90,12 @@ case class DBConfig(name: String = "sqlite-persistent",
       location = { dbp: DBParams => "jdbc:sqlite:" + dbp.dbName + ".db" },
       jdbc3 = true
     ),
-    "h2-memory" -> DBInfo(
-      description = "H2 in-memory",
-      driver = "org.h2.Driver",
-      location = { dbp: DBParams => "jdbc:h2:mem:" + dbp.dbName }
-    ),
-    "h2-persistent" -> DBInfo(
-      description = "H2 persistent",
-      driver = "org.h2.Driver",
-      location = { dbp: DBParams => "jdbc:h2:" + dbp.dbName }
-    ),
-    "postgresql" -> DBInfo(
```

Comment:
it is possible for someone to make their own `query-postgresql.sql` which has these contents. However, I hope that we have a release of the java project soon, which is more configurable, and obviates anorm.